frontend/src/services/apiService.ts

```diff
@@ -40,6 +34,20 @@ export function unwrap<T>(promise): Promise<T> {
     });
 }
 
+export function toUrlParams(obj: object): string {
```

Comment:
I think using `Array.join` would be better here

Reply:
+ Please add a quick comment to explain what it does
And give an example (highlight the fact that it does _not_ add a `&` at the beginning or the end)

Reply:
Not an array